src/migrations/dev_mode.rs

```diff
@@ -163,24 +164,39 @@ async fn get_db_migration(cli: &mut Connection) -> anyhow::Result<Option<String>
 async fn migrate_to_schema(cli: &mut Connection, ctx: &Context) -> anyhow::Result<bool> {
     use gel_protocol::server_message::TransactionState::NotInTransaction;
 
-    let transaction = matches!(cli.transaction_state(), NotInTransaction);
-    if transaction {
-        execute(cli, "START TRANSACTION", None).await?;
-    }
+    let old_timeout = timeout::inhibit_for_transaction(cli).await?;
     async_try! {
         async {
-            _migrate_to_schema(cli, ctx).await
+            if matches!(cli.transaction_state(), NotInTransaction) {
+                execute(cli, "START TRANSACTION", None).await?;
+                async_try! {
+                    async {
+                        let apply_hooks = true;
```

Comment:
What's the intention of using a constant value here? Shouldn't this be pulled from `ctx`?

Reply:
Oh, it was to give the value a name, in the lack of named parameters in Rust.